packages/fast-usdc/src/type-guards.js

```diff
@@ -147,12 +147,7 @@ export const ChainPolicyShape = {
 };
 harden(ChainPolicyShape);
 
-/**
- * @type {TypedPattern<FeedPolicy>}
- *
- * Should be JSON serializable so CLI can specify policy. E.g. no bigint,
```

Comment:
IIUC bigint is the only part that can't be JSON.

consider keeping this comment and saying "must by CopyData"